app/src/main/java/com/anzelmas/quizzando/Constants.kt

```diff
@@ -0,0 +1,134 @@
+package com.anzelmas.quizzando
+
+object Constants {
```

Comment:
No need keep all constants inside of object file. We can safely drop both the class and the object and use top-level `val`s
https://blog.egorand.me/where-do-i-put-my-constants-in-kotlin/